api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -235,6 +235,10 @@ type NicClusterPolicySpec struct {
 	NvIpam                 *NVIPAMSpec              `json:"nvIpam,omitempty"`
 	PSP                    *PSPSpec                 `json:"psp,omitempty"`
 	NicFeatureDiscovery    *NICFeatureDiscoverySpec `json:"nicFeatureDiscovery,omitempty"`
+	// Configure the directory where cni binaries would be deployed to
```

Comment:
this ends up in the description of the crd.

please add a "." at  the end of the first sentence so it reads properly.

also we do have a default today dont we ? defined today in each of the states.
all assuming `/opt/cni/bin` IIRC.

what logic is broken related to openshift ?
also i dont think the end user needs to see this in CRD definition.